src/components/Routes/Routes.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import Routes from "./Routes";
+import { MemoryRouter, Router } from "react-router-dom";
+import LoginHandler from "LoginHandler/LoginHandler";
+import DefaultRedirect from "../DefaultRedirect/DefaultRedirect";
+import { wrap } from "module";
+
+let loginHandler;
+let loginHandlerInstance;
+const mockUpdateAuthenticationInfo = jest.fn();
+
+beforeEach(() => {
+  jest.resetAllMocks();
+});
+
+it("renders Routes with 2 redirects if path is /authresponse", () => {
```

Comment:
Why? What does 2 redirects mean?
One of the benefits of unit testing is 'automatic documentation' and we get that by documenting the tests properly. Why are you testing that there are 2 redirects?

Reply:
Updated test names.